<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APISettingsDialog</class>
 <widget class="QDialog" name="APISettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ğŸ”‘ API AyarlarÄ±</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#fcd535;&quot;&gt;MEXC API AnahtarlarÄ±&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInfo">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#848e9c;&quot;&gt;API anahtarlarÄ±nÄ±z ÅŸifreli bir ÅŸekilde saklanÄ±r. Binance hesabÄ±nÄ±zdan Futures Trading izinli API anahtarlarÄ± oluÅŸturun. &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#f6465d;&quot;&gt;Asla Withdrawal iznini aktif etmeyin!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupEnvironment">
     <property name="title">
      <string>Ortam SeÃ§imi</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioTestnet">
        <property name="text">
         <string>ğŸ§ª Testnet (Test OrtamÄ± - Tavsiye Edilir)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioMainnet">
        <property name="text">
         <string>ğŸ’° Mainnet (GerÃ§ek Ä°ÅŸlemler - DÄ°KKAT!)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblEnvironmentWarning">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#fcd535;&quot;&gt;âš ï¸ Ä°lk testlerinizi mutlaka Testnet'te yapÄ±n!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupMasterPassword">
     <property name="title">
      <string>ğŸ” Master Password (Åifreleme AnahtarÄ±)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_masterpass">
      <item>
       <widget class="QLabel" name="lblMasterPasswordInfo">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#848e9c;&quot;&gt;API anahtarlarÄ±nÄ±zÄ± ÅŸifrelemek iÃ§in gÃ¼Ã§lÃ¼ bir ÅŸifre belirleyin. Bu ÅŸifreyi unutursanÄ±z API anahtarlarÄ±nÄ±z kurtarÄ±lamaz!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_masterpass">
        <item row="0" column="0">
         <widget class="QLabel" name="lblMasterPassword">
          <property name="text">
           <string>Åifre:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_masterpass">
          <item>
           <widget class="QLineEdit" name="lineEditMasterPassword">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>36</height>
             </size>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>GÃ¼Ã§lÃ¼ bir ÅŸifre belirleyin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnToggleMasterPasswordVisibility">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>36</height>
             </size>
            </property>
            <property name="toolTip">
             <string>GÃ¶ster/Gizle</string>
            </property>
            <property name="text">
             <string>ğŸ‘ï¸</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblConfirmPassword">
          <property name="text">
           <string>Åifre Tekrar:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditConfirmPassword">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Åifreyi tekrar girin</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_strength">
        <item>
         <widget class="QLabel" name="lblPasswordStrengthLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Åifre GÃ¼cÃ¼:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressPasswordStrength">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximum">
           <number>4</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPasswordStrengthText">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#848e9c;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupAPIKeys">
     <property name="title">
      <string>API AnahtarlarÄ±</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>16</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblAPIKey">
        <property name="text">
         <string>API Key:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditAPIKey">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>MEXC API Key'inizi buraya girin</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSecretKey">
        <property name="text">
         <string>Secret Key:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_secret">
        <item>
         <widget class="QLineEdit" name="lineEditSecretKey">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>MEXC Secret Key'inizi buraya girin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnToggleSecretVisibility">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>36</height>
           </size>
          </property>
          <property name="toolTip">
           <string>GÃ¶ster/Gizle</string>
          </property>
          <property name="text">
           <string>ğŸ‘ï¸</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupStatus">
     <property name="title">
      <string>BaÄŸlantÄ± Durumu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_status">
        <item>
         <widget class="QLabel" name="lblConnectionStatusIcon">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>ğŸ”´</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblConnectionStatusText">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#848e9c;&quot;&gt;BaÄŸlantÄ± test edilmedi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnTestConnection">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>ğŸ”„ BaÄŸlantÄ±yÄ± Test Et</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditConnectionLog">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>BaÄŸlantÄ± test sonuÃ§larÄ± burada gÃ¶rÃ¼necek...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblSecurityNote">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#fcd535;&quot;&gt;ğŸ”’ GÃ¼venlik Notu: API anahtarlarÄ±nÄ±z AES-256 ÅŸifreleme ile korunur ve sadece bu bilgisayarda saklanÄ±r.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnDeleteKeys">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>ğŸ—‘ï¸ AnahtarlarÄ± Sil</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Ä°ptal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>ğŸ’¾ Kaydet</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>APISettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
